utils/generate-stack.sh

```diff
@@ -16,7 +16,13 @@ mkdir "$NEW_DIR_PATH"
 # copy files
 cp -r "$ROOT_DIR"/utils/stack-templates/* "$NEW_DIR_PATH"
 
+UNAME=$(uname)
+
 for file in "$NEW_DIR_PATH"/*
 do
-  sed -i '' "s/{{STACK_NAME}}/$STACK_NAME/g" $file
+  if [ "$UNAME" = "Darwin" ] ; then
+    sed -i '' "s/{{STACK_NAME}}/$STACK_NAME/g" "$file"
+  else
+    sed -i "s/{{STACK_NAME}}/$STACK_NAME/g" "$file"
+  fi
```

Comment:
It's too bad we have to do this but unfortunately the `-i` is treated definitely on the different OS's.